src/collections/resources/articles/gitops/what-is-gitops.mdx

```diff
@@ -18,38 +18,63 @@ import { ResourcesWrapper } from "../../Resources.style.js";
-
-### 2) Versioned and Immutable
-Desired state is stored in a way that enforces immutability, versioning and retains a complete version history.
-
-### 3) Pulled Automatically
-Software agents automatically pull the desired state declarations from the source.
-
-### 4) Continuously Reconciled
-Software agents continuously observe actual system state and attempt to apply the desired state.
+ <p> According to this principle, the entire system should have a declarative description. Let us first understand what a system description is. What is committed to your Git repository is called the System Description. One or more files that define each system component and its state will be included in the system description. According to GitOps, the way in which we store those definitions is crucial, and we must do so declaratively. That implies that the description of our system will be saved as data. 
+</p>
+ 
+ <p> In the declarative approach, we specify how we want the system to look not how we can achieve that state. If we want to make any changes, we change the description instead of the series of steps to get there. Declarative configuration is critical for GitOps because it provides a description of the system that an automated agent can understand and utilize to take action. 
+ </p>
 
-OpenGitOps is a set of open-source standards, best practices, and community-focused education to help organizations adopt a structured, standardized approach to implementing GitOps.
+### 2) Single Source of Truth
+ <p> The second principle mandates that we keep that system description inside of Git. Therefore, we decide to maintain the official blueprints, which outline the ideal system state version in Git. A git commit is required if we wish to modify the blueprint. The blueprint can also be called the desired state. This helps developers, testers, operations, security, and automations to have a single reference and keep uniformity in everyone’s vision. 
+ </p>
+ 
+ <p> GitOps also improves a system's ability to recover from failure because it's simple to roll back an unsuccessful change or restore the entire system from the repository.
+</p>
 
-If you are new to GitOps, it builds and iterates on ideas drawn from DevOps and Infrastructure as Code that started with Martin Fowler’s comprehensive Continuous Integration overview and provides the freedom to choose the tools that you need for your specific use cases.
+### 3) Automated Change Delivery
+ <p> Only automation allows us to apply modifications made to the blueprint to systems already in operation. Delivery of changes is entirely automatic. GitOps doesn't allow manual editing. Because standard workflows only need Git, which is such a well-known platform, automation enables changes to be delivered through simpler for developers to use workflows. Additionally, automation standardizes your delivery processes, improving the predictability and consistency of system operations. 
```

Comment:
Git or github ? 
as git is not a platform it is  a DevOps tool used for source code management. Automation is done through github